Makefile

```
@@ -132,7 +133,7 @@ publish-image-e2e: image-e2e
 	docker push $(ARO_IMAGE)
 
 extract-aro-docker:
-	hack/ci-utils/extractaro.sh ${ARO_IMAGE}
+	hack/ci-utils/extractaro.sh $${ARO_IMAGE:-}
```

I think this is where your e2e failure is coming from:
```suggestion
	hack/ci-utils/extractaro.sh $${ARO_IMAGE}
```
~~~
+ DOCKERID=
make: *** [Makefile:136: extract-aro-docker] Error 1
~~~

---

I was able to fix the e2e pipeline by hardcoding `ARO_IMAGE` here: https://github.com/Azure/ARO-RP/blob/ce4e22ae71e5d2886db1cdba5857ef19e067a682/.pipelines/e2e.yml#L47